Dockerfile

```diff
@@ -16,11 +26,11 @@ ARG BUILD_TYPE='dev'
 
 RUN if [ "${BUILD_TYPE}" = "dev" ]; then \
         if [ "${PYTHON_VERSION}" \< "3.6" ]; then \
-            pip install --no-cache-dir -e '.[doctest, document, example, testing]' -f https://download.pytorch.org/whl/torch_stable.html; \
+            pip install --no-cache-dir --progress-bar off -e '.[doctest, document, example, testing]' -f https://download.pytorch.org/whl/torch_stable.html; \
```

Comment:
How about putting the common pip install options in a variable and reuse it to DRY the commands?

```
PIP_OPTIONS="--cache-dir --progress-bar off"

pip install $PIP_OPTIONS -e '.[checking, ...]'
pip install $PIP_OPTIONS -e .
pip install $PIP_OPTIONS -e jupyter notebook
```

Reply:
Sounds good!